include/pmacc/mappings/kernel/MappingDescription.hpp

```diff
@@ -76,9 +76,12 @@ class MappingDescription
             {
                 minBlock = std::min(minBlock, gridSuperCells[2]);
             }
-            PMACC_VERIFY(
+            PMACC_VERIFY_MSG((
                 ( guardingSuperCells[ d ] == 0 && gridSuperCells[ d ] >= 1) ||
-                gridSuperCells[ d ] >= 2 * guardingSuperCells[ d ] + 1
+                gridSuperCells[ d ] >= 2 * guardingSuperCells[ d ] + 1 ),
+                "d=" + std::to_string( d )  + "guardingSuperCells[ d ] = " +
+                std::to_string( guardingSuperCells[ d ] )  + "gridSuperCells[ d ] = " +
+                std::to_string( gridSuperCells[ d ] )
```

Comment:
Could you provide this change as a separate PR? I love it, just it's not connected to the plugin.

Reply:
Oh I didn't even notice it is still here. A separate PR is fine with me.

Reply:
@sbastrakov This should be actually easy to verify without this extra information just by looking at the param files. I run into this only while trying to initialize a dummy `MappingDescription` with (0,0,0) SC size. So maybe there is no need to include it at all.

Reply:
Yes, but I think having the message is still an improvement. Could also wait for @psychocoderHPC 's opinion.

Reply:
I opened a pull request. We can always continue to discuss it there.